### PR TITLE
Add anagram_search.java

### PR DESCRIPTION
**Pull request by ritwik12**
Implemented  issue #909 in java

### CODE REVIEW

File: code/string-algorithms/anagram_search/anagram_search.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/* Part of Cosmos by OpenGenus Foundation */
+import java.util.Arrays;  
+import java.util.Scanner;  
+       
+    public class anagram_search {
```
Comment (yatharthshahjpr) [inline code/string-algorithms/anagram_search/anagram_search.java]: Code is pretty cool 👍      
But the  class name does not follow the camel casing rule ......kindly look into it .......Well by changing the class name you will have to change the file name also !
Please write class name as  :-     Anagram_Search 
Please write file name as :-      Anagram_Search.java ! 👍

Comment (arnavb) [reply code/string-algorithms/anagram_search/anagram_search.java]: It doesn't have to be camel case, it just has to be consistent (Which it is not)

Comment (yatharthshahjpr) [reply code/string-algorithms/anagram_search/anagram_search.java]: I don't know exactly what is meant by consistent here..........but still https://www.javatpoint.com/java-naming-conventions this is a good reference ! 👍 ...may kindly check ! :)

Comment (arnavb) [reply code/string-algorithms/anagram_search/anagram_search.java]: @yatharthshahjpr By consistency I meant that in some cases Camel Case is used, while in other underscore case. The author needs to pick one.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

File: code/string-algorithms/anagram_search/anagram_search.java (hunk 1 above)
Comment (ritwik12) [reply code/string-algorithms/anagram_search/anagram_search.java]: Any conclusion guys, changes required?? @yatharthshahjpr  and @arnavb

**Pull request merged**